knowledge-base/reusing-itemtemplate-listpicker-dotnet-maui.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Reusing ItemTemplate with ListPicker in .NET MAUI
+description: Learn how to reuse a single ItemTemplate across multiple ListPickers in .NET MAUI using a value converter for dynamic data binding.
+type: how-to
+page_title: Reusing ItemTemplate Across ListPickers in .NET MAUI
+meta_title: Reusing ItemTemplate Across ListPickers in .NET MAUI
+slug: reusing-itemtemplate-listpicker-dotnet-maui
+tags: listpicker, .net maui, itemtemplate, displaymemberpath, converter
+res_type: kb
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 11.0.0 | Telerik UI for .NET MAUI ListPicker | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova) | 
+
+## Description
+
+I want to reuse a single ItemTemplate across multiple instances of the [ListPicker]({%slug listpicker-overview%}) in my .NET MAUI application. Each ListPicker uses the `DisplayMemberPath` property but displays different types of data. I want to avoid manually overriding the `ItemTemplate` for each RadListPicker and instead define an implicit `ItemTemplate` that uses a value converter to display the correct data based on the ListPicker's data source.
```

Comment:
```suggestion
I want to reuse a single ItemTemplate across multiple instances of the [ListPicker]({%slug listpicker-overview%}) in my .NET MAUI application. Each ListPicker uses the `DisplayMemberPath` property but displays different types of data. I want to avoid manually overriding the `ItemTemplate` for each `RadListPicker` and instead define an implicit `ItemTemplate` that uses a value converter to display the correct data based on the ListPicker's data source.
```